InformationScripting/scripts/sink.py

```diff
@@ -0,0 +1 @@
+ 
```

Comment:
Do you really mean less overhead (starting a python environment and 'running' this script) or less implementation (done once and trivial in C++)? Or do you mean less overhead from python scripts?

BTW what's an example where we would need this?